Add `Resurrection Cup 2024` tournament and listing. (#12582)
ppy/osu-wiki: wiki/Tournaments/RESC/2024/en.md

---
tags:
  - ResCup
  - Res Cup
  - Serenity
---

# Resurrection Cup 2024

![Resurrection Cup 2024](img/banner.jpg)

**Resurrection Cup 2024** was a 4v4, double-elimination osu! tournament hosted by ::{ flag=HU }:: [Phreel](https://osu.ppy.sh/users/12840110) and ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512). It was the third instalment in the Resurrection Cup series.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-05-11/2024-06-01 |
| Screening phase | 2024-06-01/2024-06-08 |
| Qualifiers showcase | 2024-06-09 |
| Qualifiers | 2024-06-14/2024-06-16 |
| Round of 32 | 2024-06-21/2024-06-23 |
| Round of 16 | 2024-06-28/2024-06-30 |
| Quarterfinals | 2024-07-05/2024-07-07 |
| Semifinals | 2024-07-12/2024-07-14 |
| Finals | 2024-07-19/2024-07-21 |
| Grand Finals | 2024-07-26/2024-07-28 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 70% of the prizepool, exclusive profile banner |
> | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the prizepool, exclusive profile banner |
> | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10% of the prizepool, exclusive profile banner |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 70% of the prize pool, exclusive profile banner |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the prize pool, exclusive profile banner |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10% of the prize pool, exclusive profile banner |
```

> | 4th place | Exclusive profile banner |

Reviewer: not sure if this is right tbh, i can see this done most recently on the [COE wiki page](https://osu.ppy.sh/wiki/en/Tournaments/COE/COE_2024)
```suggestion
| "4th place" | Exclusive profile banner |
```

![](img/badge.png "Resurrection Cup 2024 winner badge")

## Organisation

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=HU }:: [Phreel](https://osu.ppy.sh/users/12840110), ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512) |
| Lead GFX | ::{ flag=ID }:: [Len_licht](https://twitter.com/Len_licht) |

> | Team Serenity | ::{ flag=GB }:: [Ethereal_Winter](https://osu.ppy.sh/users/9780417), ::{ flag=ID }:: [Len_licht](https://twitter.com/Len_licht), ::{ flag=FR }:: [Mimiliaa](https://osu.ppy.sh/users/7117621), ::{ flag=CA }:: [RedcXca](https://osu.ppy.sh/users/14056601), ::{ flag=HU }:: [Yumeyo](https://twitter.com/Yumeyo_art) |

Author: Updated

> | Team Serenity | ::{ flag=GB }:: [Ethereal_Winter](https://osu.ppy.sh/users/9780417), ::{ flag=ID }:: [Len_licht](https://twitter.com/Len_licht), ::{ flag=FR }:: [Mimiliaa](https://osu.ppy.sh/users/7117621), ::{ flag=CA }:: [RedcXca](https://osu.ppy.sh/users/14056601), ::{ flag=HU }:: [Yumeyo](https://twitter.com/Yumeyo_art) |

Reviewer 2: Yeah, Team Serenity is a group that consists of all the people who animated her, drawn her and voiced her.

Her lore is not being done by this team

| **im actually crying right now** | **5** | 2 | fajny bambule | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114409760) |
| **ALL-IN IÑAKI** | **5** | 0 | super hamburger | *win by default* |
| **rich penguins** | **5** | 2 | cute kittens | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114411190) |
| **Nipis Madu** | **5** | 1 | DJ Ramon Sucesso | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114411923) |
| **fresh off the boat** | **5** | 0 | doge | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114412691) |
| **Grease Monkey** | **5** | 1 | Gente Seria: die Fortsetzung | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114413476) |
| **Financial Advisors** | **5** | 0 | el hambre y las ganas de comer | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114414384) |
| **<< TIDAL WAVE >>** | **5** | 3 | Cowabunga | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114414350) |

### Qualifiers

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Reviewer: ```suggestion
Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1, where 1 is the best performing team, and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).
```

Author: Being discussed in Wala's review

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Author: The problem here, is that it's not solely Z-sum, but they're placed on a scale between 0 and 1. Not reflecting that on the wiki seems extremely wrong to me, and would also violate official support rules. Though I will admit, I struggled with wording this part properly.

To resolve the Z-sum being confusing for those who are not already aware of what it means, I can add the definition of Z-score and mention it's the sum of those.
```suggestion
Seeding results are calculated by taking the sum of [Z-scores](https://en.wikipedia.org/wiki/Standard_score) of the teams and placing those linearly between 0 and 1, where 1 is the best performing team, and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).
```

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Reviewer 3: linear scaling preserves ordering, so it doesn't matter for seeding purposes. what part of support rules does it violate?

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Reviewer 3: this kind of thing can be expressed much more briefly

```suggestion
Seeding results are calculated by ranking each team by their sum of [Z-scores](https://en.wikipedia.org/wiki/Standard_score), normalised to a scale of 0 to 1. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).
```

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Author: As mentioned on the [wiki](<https://osu.ppy.sh/wiki/en/Tournaments/Official_support#:~:text=A%20third%20party%20should%20be%20able%20to%20get%20the%20same%20results%20without%20needing%20additional%20information.>)

> A third party should be able to get the same results without needing additional information.

If we leave out the normalisation part, users would not be able to get the same results, since they would need additional information, in this case the need to normalise the scores to a scale of 0-1, in order to get the same results. 

The suggestion you made works for me though, so we can go with that.

> Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).

Reviewer 3: why are we relying on text descriptions instead of formulas or code for support compliance... luckily this is not my problem

> ## Ruleset

Reviewer 3: contractions like don't and you're should be expanded in wiki articles

### General Information

> 1. All matches will be played with ScoreV2 in Head-to-head mode.

Reviewer: ```suggestion
1. All matches will be played with ScoreV2 in head-to-head mode.
```

Author: Both other wiki pages, as well as the game itself has this as "Head-to-head" wth the first h capitalised.

> 2. 32 teams will qualify for the bracket stage, chosen through Qualifiers Stage.

Reviewer: ```suggestion
2. 32 teams will qualify for the bracket stage, chosen through the Qualifiers stage.
```

3. All maps will be played with No Fail enabled, in addition to the appropriate mods.

### Registration

1. The minimum team size is 4 members, and the maximum team size is 8 members.
2. The team captain must be present in the Discord until the end of their team's tournament run. If none of the team members can be contacted by any means, it results in a forfeit.
3. Staff members and former staff members (before registrations opened) who participated in the production of this iteration are not allowed to participate, with the exception of commentators.

### Qualifiers Procedure

> 1. The referee will create the lobby around 10 minutes before the scheduled match time, ping all assigned team captains in the Discord server and send out the invites shortly after.

Reviewer: ```suggestion
1. The referee will create the lobby around 10 minutes before the scheduled match time, ping all assigned team captains in the Discord server, and send out the invites shortly after.
```

> 2. Teams have 5 minutes to gather enough players in the lobby. If a team cannot gather 4 players, they can reschedule for a different, already existing lobby. 

Reviewer: ```suggestion
2. Teams have 5 minutes to gather enough players in the lobby. If a team cannot gather 4 players, they can reschedule for a different, already existing lobby.
```

> 2. Teams have 5 minutes to gather enough players in the lobby. If a team cannot gather 4 players, they can reschedule for a different, already existing lobby. 

Reviewer: ```suggestion
2. Teams have 5 minutes to gather enough players in the lobby. If a team cannot gather 4 players, they can reschedule for a different, already existing lobby.
```

3. Each map will be played in order as they are shown on the mappool sheet. There will be one playthrough of the pool.
4. If there is a disconnect, the player(s) that disconnected will be allowed to replay the map at the end of the lobby.

### Bracket stage procedure

1. The referee will create the lobby around 10 minutes before the scheduled match time, ping both team captains in the Discord server, and send out the invites shortly after.
2. Teams have 10 minutes to gather enough players in the lobby. If a team fails to gather 4 players in the lobby on time, the match will be counted as a win for the opponent.
   - If neither of the teams show up, they will be able to reschedule to a different time. If no suitable time can be found, the team with the higher qualifier seeding will be given the advantage.
3. In case of a disconnect, a team will be able to replay the map if the disconnect happened within the first 45 seconds of the map. A team is only allowed to replay one map during the whole match, any additional disconnects will not be taken into consideration and the opponent will win the point.

>    - The referee may allow additional replays at their discretion and with confirmation from the opponent. If you're having major internet issues so you keep disconnecting every map, you should not play.

Reviewer: ```suggestion
   - The referee may allow additional replays at their discretion and with confirmation from the opponent. If you're having major internet issues causing you to keep disconnecting every map, you should not play.
```

6. Free Mod requires both teams to have a player picking HD and a player picking HR, the remaining two players can pick any of the following mods: HD, HR, FL, or EZ (x1.75). All players must have a mod.
7. During the Tiebreaker, Free Mod is allowed without limitations, meaning you can pick a mod if you feel like it, but it is not required. Permitted mods are HD, HR, FL & EZ.

### Stage information

| Stage | Best of | Ban count |
| :-- | --: | --: |
| Qualifiers | 1[^qualifier-playthrough] |  |
| Round of 32 | 9 | 1 |
| Round of 16 | 9 | 1 |
| Quarterfinals | 11 | 2 |
| Semifinals | 11 | 2 |
| Finals | 13 | 2 |
| Grandfinals | 13 | 2 |

### Mappools

| Stage | Star Rating | NM | HD | HR | DT | FM | TB |
| :-- | --: | --: | --: | --: | --: | --: | --: |
| Qualifiers | 7.5★ | 4 | 3 | 3 | 3 |  |  |
| Round of 32 | 6.7★ | 4 | 2 | 2 | 3 | 3 | 1 |
| Round of 16 | 7.0★ | 4 | 2 | 2 | 3 | 3 | 1 |
| Quarterfinals | 7.3★ | 5 | 3 | 3 | 4 | 4 | 1 |
| Semifinals | 7.5★ | 5 | 3 | 3 | 4 | 4 | 1 |
| Finals | 7.8★ | 5 | 3 | 3 | 4 | 4 | 1 |
| Grandfinals | 8.0★ | 5 | 3 | 3 | 4 | 4 | 1 |

### Scheduling

#### Qualifiers

> 1. Qualifier lobbies can be scheduled in the Discord server in the #scheduling channel. Rescheduling a qualifier lobby can be done here as well.

Reviewer: ```suggestion
1. Qualifier lobbies can be scheduled in the Discord server in the `#scheduling` channel. Rescheduling a qualifier lobby can be done here as well.
```

2. If a team doesn't show up to their qualifiers lobby, they may reschedule to any other lobby provided there is free space in that lobby. The hosts should be contacted if this is the case. If the team is able to play in the very next lobby, and there is space for them to do so, they should additionally contact the referee of that lobby.

> 3. If a team fails to show up to any qualifiers lobby, they will be disqualified. 

Reviewer: ```suggestion
3. If a team fails to show up to any qualifiers lobby, they will be disqualified.
```

> 3. If a team fails to show up to any qualifiers lobby, they will be disqualified. 

Reviewer: ```suggestion
3. If a team fails to show up to any qualifiers lobby, they will be disqualified.
```

4. If it is not possible for a team to make any of the scheduled lobbies, they may request a custom time for their lobby subject to staff availability and host's discretion.

#### Bracket stage

> 1. If the assigned time does not work with your team's schedule/you don't like it, please reschedule your match using the format from the pinned messages.

Reviewer: ```suggestion
1. If the assigned time does not work with your team's schedule/you don't like it, please reschedule your match using the format found pinned in the `#scheduling` channel on Discord.
```

> 2. Reschedules will be accepted until 12hrs before the original time of the match, and only if you reschedule to a later time. If you reschedule to an earlier time, reschedules will only be accepted until 12hrs before the new time of the match.
> 3. If you post a reschedule later than 12hrs before the original/new time, approval is subject to staff availability and host's discretion. If no referee can be found for that time, the reschedule may be declined and your match will be played at the original time. Attempts at rescheduling to a different time are permitted.

Reviewer: ```suggestion
2. Reschedules will be accepted until 12 hours before the original time of the match, and only if you reschedule to a later time. If you reschedule to an earlier time, reschedules will only be accepted until 12 hours before the new time of the match.
3. If you post a reschedule later than 12 hours before the original/new time, approval is subject to staff availability and host's discretion. If no referee can be found for that time, the reschedule may be declined and your match will be played at the original time. Attempts at rescheduling to a different time are permitted.
```
